src/styles/styles.js

```diff
@@ -755,12 +755,60 @@ const styles = {
         borderRightWidth: 0,
     },
 
-    chatSwitcherAvatar: {
-        backgroundColor: themeColors.icon,
-        borderRadius: 20,
+    chatSwitcherSingleAvatar: {
+        height: variables.avatarSizeNormal,
+        width: variables.avatarSizeNormal,
+        borderRadius: variables.avatarSizeNormal,
+    },
+
+    chatSwitcherBigAvatar: {
         height: variables.componentSizeNormal,
-        overflow: 'hidden',
         width: variables.componentSizeNormal,
+        borderRadius: variables.componentSizeNormal,
+    },
+
+    chatSwitcherAvatar1: {
+        marginTop: -variables.componentSizeNormal / 2,
+        borderRadius: variables.avatarSizeNormal,
+    },
+
+    chatSwitcherAvatar2: {
+        marginTop: variables.avatarSizeNormal / 2,
+        height: variables.avatarSizeNormal + 6,
+        width: variables.avatarSizeNormal + 6,
+        borderRadius: variables.avatarSizeNormal + 6,
+        left: -variables.avatarSizeNormal / 2,
+    },
+
+    chatSwitcherAvatarText: {
+        backgroundColor: colors.gray3,
```

Comment:
```suggestion
        backgroundColor: themeColors.icon,
```

Reply:
Ok.

Reply:
I showed all icons of DM, 3 participants, and 4+ participants.
Do you need a table of screenshots instead of the above?

Reply:
Nope, I think what you've provided is good enough!